lib/capistrano/tasks/nginx.rake

```diff
@@ -13,6 +13,9 @@ namespace :load do
     # ssl options
     set :nginx_location, '/etc/nginx'
     set :nginx_use_ssl, false
+    set :nginx_use_spdy, false
```

Comment:
You wanted to include this line in this pull request? I think the setting makes sense but maybe add it in a separate commit. Feel free to push it straight to master.